donate.html

```diff
@@ -0,0 +1,304 @@
+          </div>
+        </div>
+        <!-- Image -->
+        <div class="col-md-6" style="display:flex">
+          <div class="story-image">
+            <img src="img/2015_img.jpg" alt="Technica Hackers working hard at their hackathon projects." class="center-block">
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- Donor Promise -->
+  <div class="donate-promise">
+    <div class="container">
+      <!-- Image -->
+      <div class="row">
+        <div class="col-md-6 order-2 order-md-1" style="display:flex">
+          <div class="promise-image">
+              <img src="img/2017_img.jpg" alt="A neon light hardware hack with Technica's signature colors." class="center-block">
+          </div>
+        </div>
+        <div class="col-md-6 order-1 order-md-2">
+          <!-- Header -->
+          <h1 class="our-donor">Our Donor Promise</h1>
+            <!-- Body -->
+          <div class="promise-body">
+            <p> <b> We promise to respect your donation by only using donor funds to support our mission of reaching gender equity in tech. </b> The most high cost ways we spend funds are to rent out our hackathon venue space, providing meals to our hackers all weekend long, and ensuring safety in the form of meeting fire marshal regulations and event health personnel.</p>
+            <p> <b> Additionally, Technica is a 501(c)(3) public charity, so your donations are tax-deductible.</b></p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- jules -->
+
+  <!-- donor impact -->
+  <div class="donate-impact">
+    <div class="container">
+      <div class="text-center">
+      <!-- header -->
+        <div class="impact-header">
+          <h1>Your Donation's Impact</h1>
+        </div>
+
+      <!-- body -->
+      <div class="row" style="padding-bottom: 3%;">
+        <div class="col-sm">
+          <div class="donate-option1">
+              <h2 class="donate-option1-header">$20</h2>
+              <p>The cost to provide electricity and internet to a hacker, so they can stay charged up and logged in all weekend</p>
+          </div>
+        </div>
+        <div class="col-sm">
+          <div class="donate-option2">
+              <h2 class="donate-option2-header">$10</h2>
+              <p>The cost of each meal we provide to hackers at Technica, keeping them fueled and properly fed</p>
+          </div>
+        </div>
+        <div class="col-sm">
+          <div class="donate-option3">
+              <h2 class="donate-option3-header">$5</h2>
+              <p>The cost per hacker to keep Emergency Medical Services and Security Personnel on hand throughout Technica weekend, just in case</p>
+          </div>
+        </div>
+      </div>
+
+      <!-- donate button -->
+      <div>
+        <a target="_blank" href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=STSQQD9BAPZSJ&source=url" class="donate-button">Donate</a>
```

Comment:
i can't find this class in the css, but i think this would look better slightly shorter and/or longer - reference figma file for dimensions

Reply:
Oh...I did use the dimensions from the figma file. Ill take a look at it again